bin/polyglot

```diff
@@ -284,13 +288,13 @@ IFS="$OIFS"
 for language in ${langs[@]}; do
     file="$langs_path/$language.lproj/$file_to_translate"
     echo ''
-    info Processing $file
-    file_lines=`grep -c "=" "$file"`
+    info "Processing $file"
+    file_lines=`get_strings_from_file $file | wc -l`
     new_strings=true
     content_diff_reverse=''
 
     if [ $file_lines -eq $base_file_lines ]; then
-        content_diff=`diff <(echo "$base_file_content") <(grep "=" "$file")`
+        content_diff=`diff <(echo "$base_file_content") <(get_strings_from_file $file)`
```

Comment:
I suggest reading the file once and reusing its contents for the further subtasks. I/O operations are much more expensive than the amount of RAM, which will be used to store its contents